sqlglot/errors.py

```diff
@@ -9,17 +9,21 @@ class ErrorLevel(AutoName):
     RAISE = auto()
 
 
-class UnsupportedError(ValueError):
+class SqlglotError(ValueError):
```

Comment:
I'm not sure if this should subclass ValueError... but removing that subclass would technically be a breaking change. So I was scared to do otherwise.